src/Algebra/Morphism/Construct/Terminal.agda

```diff
@@ -0,0 +1,98 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The unique morphism to the terminal object,
+-- for each of the relevant categories. Since
+-- each terminal algebra builds on `Monoid`,
+-- possibly with additional (trivial) operations,
+-- satisfying additional properties, it suffices to
+-- define the morphism on the underlying `RawMonoid`
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Level using (Level)
+
+module Algebra.Morphism.Construct.Terminal {c ℓ : Level} where
+
+open import Algebra.Bundles using (Monoid; Group)
+open import Algebra.Bundles.Raw using (RawMonoid; RawGroup)
+open import Algebra.Construct.Terminal {c} {ℓ}
+import Algebra.Morphism.Definitions as MorphismDefinitions
+open import Algebra.Morphism.Structures
+  using ( module MagmaMorphisms
+        ; module MonoidMorphisms
+        ; module GroupMorphisms
+        )
+open import Data.Product.Base using (_,_)
+open import Function.Definitions using (StrictlySurjective)
+import Relation.Binary.Morphism.Definitions as Definitions
+open import Relation.Binary.Morphism.Structures
+
+private
+  variable
+    a ℓa : Level
+
+
+------------------------------------------------------------------------
+-- The underlying data of the morphism
+
+module 𝕆neMorphism (M : RawMonoid a ℓa) where
```

Comment:
That module name is perhaps overly cute?

Reply:
I'd wanted to overload `𝕆ne` from `Algebra.Construct.Terminal`, but resolving the disambiguation was horrible. 
As for 'cute', sure, but when writing `Algebra.Construct.Terminal` and `Algebra.Construct.Initial`, i had been trying to write a *single* internal module definition such that all the relevant definitions could be written as `record { 𝕆ne }` etc. (it nearly worked, but not quite). 

I *do* like blackboard-bold identifiers for 'canonical' constructions, but `\b1` doesn't display in my Emacs... :-(

Reply:
Alternative: `The{Unique}Morphism`?